recipes/virtual_dataframe/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set BASENAME = "virtual_dataframe" %}
+{% set PRJ_VERSION = "v0.1rc1" %}
+{% set SETUPTOOLS_SCM_PRETEND_VERSION = PRJ_VERSION %}
+{% set PRJ_PACKAGE = BASENAME %}
+{% set PYTHON_VERSION = "3.8" %}
+{% set PYTHON_VERSION_MAX = "3.9" %}
+
+package:
+  name: "virtual_dataframe"
+  version: "{{ SETUPTOOLS_SCM_PRETEND_VERSION }}"
+
+source:
+  # See https://docs.github.com/en/rest/repos/contents#download-a-repository-archive-tar
+  url: https://github.com/pprados/virtual_dataframe/tarball/{{ PRJ_VERSION }}/virtual_dataframe.tar.gz
+  sha256: 7389764a6b2f25fb18a58fb2618fa2f68fc2c06386fdb482edc0b8c90549e358
+
+build:
+  noarch: python
+  number: 0
+  # no such file directory sur PYTHON ?
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ PRJ_VERSION }}
+  script: {{ PYTHON }} -m pip install --progress-bar off --quiet --no-input . -vv
+
+requirements:
+  build:
+    - python {{ PYTHON_VERSION }}
+    - pip
+    - setuptools
+    - setuptools_scm
```

Comment:
None of these are needed nor used

Reply:
I removed this file

Reply:
During the build process, the [setuptools_scm](https://pypi.org/project/setuptools-scm/) is used to calculate the current version from the git tag. But, to use it, I need the last version of pip.